### DISCUSSION TITLE
Live Game Server Query

### DISCUSSION THREAD

**Discussion by ZekyTheWolf** [tltneon/lgsl#153] [Show and tell]
If @tltneon dont mind, i will post it here since its based on his code.

As mentioned in issue #141 there should be seperate library for easier implementaion to CMS, so i take some time to create it.

I created [LGSQ](https://github.com/ZekyTheWolf/LGSQ/releases/latest) from [lgsl_protocol.php](https://github.com/tltneon/lgsl/blob/master/lgsl_files/lgsl_protocol.php) and its available via composer or direct download in github.

### Composer
```
composer require zekythewolf/lgsq
```

### Download source [here](https://github.com/ZekyTheWolf/LGSQ/releases/latest)

There is a work to do, but for now it should be working (games tested: TS3, CS, CSGO, Discord, Minecraft), developed on PHP 8.2.x older version where not tested.

**Comment by AssaultStars** [comment]
how do installing on webspace? :)

**Reply by ZekyTheWolf** [reply]
Should be simply download and require everything inside src folder, then use it as shown in examples.

I forget to create autoloader for those who dont use composer, i will try to fix it as soon as possible.
